HashRegistrarSimplified.sol

```diff
@@ -237,6 +238,27 @@ contract Registrar {
         }
         return len;
     }
+    
+    /** 
+     * @dev Creates a scheduled release for each name
+     * 
+     * Each name will be assigned a random date in which its auction 
+     * can be started, from 0 to 6 months
+     * 
+     * @param _hash The hash to start an auction on
+     * @param _timestamp The timestamp to query about
+     */
+     
+    function isAllowed(bytes32 _hash, uint _timestamp) constant returns (bool allowed){
```

Comment:
Could we write the inverse function, which returns the date at which an auction will be allowed, and then write `isAllowed` based on that?